Gemfile

```diff
@@ -8,4 +8,5 @@ group :development do
   gem 'dotenv'
   gem 'pry-byebug', platform: :mri
   gem 'yard'
+  gem 'rspec'
```

Comment:
We tend to declare dependencies in `3scale-api.gemspec`.

`rspec` is declared there. 

It is true, though, there are some dependencies declared in Gemfile that should be moved to `3scale-api.gemspec`.

Reply:
OK, I'll remove this. Want to move any of the others - it is spread a bit between the two files....???